<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingExtSeries</class>
 <widget class="QDialog" name="SettingExtSeries">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings curves</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QGroupBox" name="advSettingsGroupBox">
       <property name="title">
        <string>Advanced settings</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="1">
         <widget class="QLineEdit" name="numDecPlacesLineEdit"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="depthOffsetLineEdit"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="shiftLbl">
          <property name="text">
           <string>Shift:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="valueShiftLbl">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="resetShiftBtn">
          <property name="text">
           <string>Reset shift</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="shiftAxisLbl">
          <property name="text">
           <string>Shift along the time axis:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="unitLineEdit"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="shiftAxisLineEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="depthOffsetLbl">
          <property name="text">
           <string>Depth offset, cm:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="numDecPlacesLbl">
          <property name="text">
           <string>Number of decimal places:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="executeShiftAxisBtn">
          <property name="text">
           <string>Execute</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="unitLbl">
          <property name="text">
           <string>Unit:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="nameChangeLbl">
          <property name="text">
           <string>Name curve:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameChangeLineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="settingsGroupBox">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="0">
         <widget class="QCheckBox" name="seriesOnOffCheckBox">
          <property name="text">
           <string>Visibility curve</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="colorBtn">
          <property name="text">
           <string>Color curve</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="minLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="minLbl">
          <property name="text">
           <string>min:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="nameLbl">
          <property name="text">
           <string>Name:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="advSetCcheckBox">
          <property name="text">
           <string>Advanced settings</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="maxLineEdit"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="nameSeriesLbl">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="maxLbl">
          <property name="text">
           <string>max:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="applyBtn">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
